src/Sulu/Bundle/ContentBundle/Resources/public/js/components/copy-locale-overlay/main.js

```diff
@@ -227,13 +248,20 @@ define(function() {
         },
 
         /**
-         * start overlay with selects to choose src and dest languages
+         * Start overlay with selects to choose src and dest languages.
+         *
+         * @param {Object} newTranslations
+         *
+         * @return {*}
          */
-        startCopyLocalesOverlay: function() {
+        startCopyLocalesOverlay: function(newTranslations) {
+            // overwrite translations
+            translations = this.sandbox.util.extend(true, {}, translations, newTranslations);
```

Comment:
Doesn't that change the existing defaults? So when it is called later without the `newTranslation` parameter, the defaults will be overridden?